mnd/mnd/forms/mnd_registration_form.py

```diff
@@ -57,3 +68,37 @@ def _clean_fields(self):
                       'patient_insurance_ndis_coordinator_email']:
                 self.fields[f].required = ndis_number_set
         super()._clean_fields()
+
+
+class MNDCarerRegistrationForm(RegistrationForm):
+    placeholders = {
+        'username': _("Username"),
+        'password1': _("Password"),
+        'password2': _("Repeat Password"),
+        'token': _("Registration token")
+    }
+
+    password_fields = ['password1', 'password2']
+
+    token = CharField(required=True, min_length=36, max_length=36)
+    registry_code = CharField(required=True)
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.setup_fields()
+
+    def setup_fields(self):
+        for field in self.fields:
+            self.fields[field].widget.attrs['class'] = 'form-control'
+            self.fields[field].widget.attrs['placeholder'] = self.placeholders.get(field, _(''))
```

Comment:
Not sure if we have to translate `''`, I'm guessing it will be `''` in all languages :-)

Reply:
Updated.